File: module07/module39/module2/module2/src/main/java/com/android/example/module07_module39_module2_module2/ClassAAD.java
package com.android.example.module07_module39_module2_module2;

public class ClassAAD {
	private 	com.android.example.module07_module39_module2_module1.ClassAAF instance_var_1_0 = new com.android.example.module07_module39_module2_module1.ClassAAF();
	private 	com.android.example.module07_module39_module2_module1.ClassAAC instance_var_1_1 = new com.android.example.module07_module39_module2_module1.ClassAAC();
	private 	com.android.example.module07_module39_module2_module1.ClassAAG instance_var_1_2 = new com.android.example.module07_module39_module2_module1.ClassAAG();
	private 	com.android.example.module07_module39_module2_module5.ClassAAI instance_var_1_3 = new com.android.example.module07_module39_module2_module5.ClassAAI();
	private 	com.android.example.module07_module39_module2_module5.ClassAAE instance_var_1_4 = new com.android.example.module07_module39_module2_module5.ClassAAE();

	public void method0(
		com.android.example.module07_module39_module2_module5.ClassAAI param0,
		com.android.example.module07_module39_module2_module1.ClassAAI param1,
		com.android.example.module07_module39_module2_module1.ClassAAD param2,
		com.android.example.module07_module39_module2_module5.ClassAAB param3) throws Throwable {
		if (new java.lang.Object().equals(new java.lang.Object())) {
			io.reactivex.subscribers.TestSubscriber<Object> local_var_3_0 = new io.reactivex.subscribers.TestSubscriber();
			local_var_3_0.onError(new java.lang.Throwable());

		} else {
			io.reactivex.subscribers.TestSubscriber<Object> local_var_3_0 = new io.reactivex.subscribers.TestSubscriber();
			local_var_3_0.dispose();

		}
		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_4 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_4.onNext(new java.lang.Object());

		param0.method2(new io.reactivex.schedulers.TestScheduler(), new io.reactivex.subscribers.TestSubscriber(), new io.reactivex.internal.disposables.ListCompositeDisposable());

	}

	public void method1(
		com.android.example.module07_module39_module2_module5.ClassAAI param0,
		com.android.example.module07_module39_module2_module5.ClassAAB param1,
		com.android.example.module07_module39_module2_module1.ClassAAG param2,
		com.android.example.module07_module39_module2_module5.ClassAAD param3) throws Throwable {
		io.reactivex.internal.disposables.ListCompositeDisposable local_var_2_4 = new io.reactivex.internal.disposables.ListCompositeDisposable();
		local_var_2_4.isDisposed();

	}

	public void method2(
		com.android.example.module07_module39_module2_module1.ClassAAF param0,
		com.android.example.module07_module39_module2_module5.ClassAAC param1) throws Throwable {
		param0.method0(new com.android.example.module07_module39_module2_module5.ClassAAI(), new com.android.example.module07_module39_module2_module5.ClassAAH());

	}

	public void method3(
		com.android.example.module07_module39_module2_module1.ClassAAH param0,
		com.android.example.module07_module39_module2_module1.ClassAAB param1,
		com.android.example.module07_module39_module2_module5.ClassAAE param2) throws Throwable {
		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_3 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_3.onNext(new java.lang.Object());

		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_4 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_4.dispose();

	}
}

File: module06/module284/module3/src/main/java/com/android/example/module06_module284_module3/ClassAAD.kt
package com.android.example.module06_module284_module3

@Suppress("UNUSED_PARAMETER")
class ClassAAD {
	private 	val instance_var_1_0: com.android.example.module06_module284_module5.ClassAAC = com.android.example.module06_module284_module5.ClassAAC()
	private 	val instance_var_1_1: com.android.example.module06_module284_module2.ClassAAD = com.android.example.module06_module284_module2.ClassAAD()
	private 	val instance_var_1_2: com.android.example.module06_module284_module5.ClassAAC = com.android.example.module06_module284_module5.ClassAAC()
	private 	val instance_var_1_3: com.android.example.module06_module284_module5.ClassAAC = com.android.example.module06_module284_module5.ClassAAC()

	fun method0(
		param0: com.android.example.module06_module284_module2.ClassAAJ,
		param1: com.android.example.module06_module284_module5.ClassAAI,
		param2: com.android.example.module06_module284_module5.ClassAAF,
		param3: com.android.example.module06_module284_module2.ClassAAF
	) {
		val local_var_2_4: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_4.isDisposed()

		if (kotlin.Any().equals(kotlin.Any())) {
			if (kotlin.Any().equals(kotlin.Any())) {
				val local_var_4_0: io.reactivex.internal.disposables.ListCompositeDisposable = io.reactivex.internal.disposables.ListCompositeDisposable()
				local_var_4_0.clear()

			}
		} else {
			val local_var_3_0: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
			local_var_3_0.onNext(kotlin.Any())

		}
		val local_var_2_5: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_5.assertSubscribed()

		val local_var_2_6: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_6.dispose()

	}

	fun method1(
		param0: com.android.example.module06_module284_module5.ClassAAH,
		param1: com.android.example.module06_module284_module5.ClassAAJ
	) {
	}

	fun method2(
		param0: com.android.example.module06_module284_module5.ClassAAH,
		param1: com.android.example.module06_module284_module2.ClassAAE,
		param2: com.android.example.module06_module284_module2.ClassAAG
	) {
		param0.method3(io.reactivex.subscribers.TestSubscriber())

	}

	fun method3(
		param0: com.android.example.module06_module284_module5.ClassAAG
	) {
		val local_var_2_1: io.reactivex.internal.schedulers.SingleScheduler = io.reactivex.internal.schedulers.SingleScheduler()
		local_var_2_1.shutdown()

		val local_var_2_2: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_2.request(4574373143371199271)

		val local_var_2_3: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_3.hasSubscription()

		param0.method0(io.reactivex.subscribers.TestSubscriber())

	}

	fun method4(
		param0: com.android.example.module06_module284_module5.ClassAAJ,
		param1: com.android.example.module06_module284_module5.ClassAAI,
		param2: com.android.example.module06_module284_module2.ClassAAB,
		param3: com.android.example.module06_module284_module2.ClassAAH
	) {
		val local_var_2_4: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_4.requestMore(-6467186231062054885)

		val local_var_2_5: io.reactivex.internal.schedulers.SingleScheduler = io.reactivex.internal.schedulers.SingleScheduler()
		local_var_2_5.shutdown()

		val local_var_2_6: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_6.request(-4634374332051522319)

		val local_var_2_7: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_7.isCancelled()

	}
}
